docs/docs/dependency-specification.md

```diff
@@ -98,6 +98,12 @@ flask = { git = "https://github.com/pallets/flask.git", rev = "38eb5d3b" }
 numpy = { git = "https://github.com/numpy/numpy.git", tag = "v0.13.2" }
 ```
 
+To use an SSH connection, for example in the case of private repositories, simply follow the standard git syntax:
```

Comment:
May I suggest a slightly-different phrasing here?
> To use an SSH connection, for example in the case of private repositories, use the following example syntax:

Reply:
This more general phrase makes sense! Thank you 💫